lib/routes/emails.js

```diff
@@ -491,12 +494,15 @@ module.exports = (log, db, mailer, config, customs, push) => {
         const sessionToken = request.auth.credentials
         const uid = sessionToken.uid
 
-        if (! features.isSecondaryEmailEnabled(sessionToken.email)) {
-          return reply(error.featureNotEnabled())
-        }
+        return db.account(uid)
+          .then((account) => {
 
-        db.accountEmails(uid)
```

Comment:
Again, I don't have all the context here so apologies if this is stupid or pointless, but is there an argument for joining with the `accounts` table in the `accountEmails` stored procedure and including `account.email` in that result?